.github/workflows/pull-requests.yml

```diff
@@ -0,0 +1,96 @@
+name: PR Checks
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  validations:
+    name: Run project validations
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        env:
+          cache-name: lagom-dependencies-cache-
+        with:
+          path: |
+            - $HOME/.cache/coursier
+            - $HOME/.ivy2/cache
+            - $HOME/.sbt
+            - $HOME/.m2/repositories
+          key: lagom-dependencies-cache-${{ hashFiles('**/*.sbt') }}
+          restore-keys: lagom-dependencies-cache-
+      - uses: AdoptOpenJDK/install-jdk@v1
+        name: Install AdoptOpenJDK 11
+        with:
+          version: "11"
+      - name: Debug env
+        run: env
+      - name: Code validations (format, binary compatibilty, whitesource, etc.)
+        run:  bin/test-code-style
+
+  run-tests:
+    name: Tests / Scala ${{ matrix.SCALA_VERSION }} / Java ${{ matrix.JDK }}
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - JDK: 8
+            SCALA_VERSION: "2.12.13"
+          - JDK: 11
+            SCALA_VERSION: "2.12.13"
+          - JDK: 11
+            SCALA_VERSION: "2.13.5"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        env:
+          cache-name: lagom-dependencies-cache-
+        with:
+          path: |
+            - $HOME/.cache/coursier
+            - $HOME/.ivy2/cache
+            - $HOME/.sbt
+            - $HOME/.m2/repositories
+          key: lagom-dependencies-cache-${{ hashFiles('**/*.sbt') }}
+          restore-keys: lagom-dependencies-cache-
+      - uses: AdoptOpenJDK/install-jdk@v1
+        name: Install AdoptOpenJDK ${{ matrix.JDK }}
+        with:
+          version: ${{ matrix.JDK }}
+      - name: Run tests
+        run: SCALA_VERSION=${{ matrix.SCALA_VERSION }} bin/test-2.1x
+      - name: Run multi-jvm tests
+        run: SCALA_VERSION=${{ matrix.SCALA_VERSION }} bin/test-multi-jvm-2.1x
+
+  run-documentation-tests:
+    name: Docs / Tests / Java ${{ matrix.JDK }}
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - JDK: 11
+          - JDK: 8
+    steps:
+      - name: "Documentation validations and tests"
+        run: bin/test-documentation
+
+  run-build-tools-tests:
+    name: Build Tools / Tests / ${{ matrix.JDK }}
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - JDK: 11
+          - JDK: 8
+    steps:
+      - name: "Scripted tests for sbt 1"
+        run: SCALA_VERSION=2.12.13 bin/test-sbt
+      - name: "PublishM2 and test Maven"
+        run: bin/test-maven
```

Comment:
Let's add a comment here (or maybe a new issue) stating this needs an extra job (a CRON one) to test with latest Akka artifacts (and a link to the code in `bin/scriptLib`).